src/snapred/backend/data/LocalDataService.py

```diff
@@ -103,70 +100,33 @@ def _readInstrumentParameters(self) -> Dict[str, Any]:
 
     def readStateConfig(self, runId: str) -> StateConfig:
         reductionParameters = self._readReductionParameters(runId)
+        diffCalRecord = self.readCalibrationRecord(runId)
+        if diffCalRecord is None:
+            diffCalRecord = self.readCalibrationState(runId)
+            particleBounds = diffCalRecord.instrumentState.particleBounds
+        else:
+            diffCalRecord = diffCalRecord.calibrationFittingIngredients
+            particleBounds = diffCalRecord.calibrationFittingIngredients.instrumentState.particleBounds
+        stateId, _ = self._generateStateId(runId)
 
         return StateConfig(
-            diffractionCalibrant=self._readDiffractionCalibrant(runId),
-            emptyInstrumentRunNumber=reductionParameters["VBRun"][0],
-            normalizationCalibrant=self._readNormalizationCalibrant(runId),
-            geometryCalibrationFileName=None,  # TODO: missing, reductionParameters['GeomCalFileName'],
-            calibrationAuthor=reductionParameters.get("calibBy"),
-            calibrationDate=reductionParameters.get("calibDate"),
+            calibration=diffCalRecord,
             focusGroups=self._readFocusGroups(runId),
             isLiteMode=True,  # TODO: Support non lite mode
             rawVanadiumCorrectionFileName=reductionParameters["rawVCorrFileName"],
             vanadiumFilePath=str(
                 self.instrumentConfig.calibrationDirectory
                 / "Powder"
-                / reductionParameters["stateId"]
+                / stateId
                 / reductionParameters["rawVCorrFileName"]
             ),
-            calibrationMaskFileName=reductionParameters.get("CalibrationMaskFilename"),
-            stateId=reductionParameters["stateId"],
-            tofBin=reductionParameters["tofBin"],
-            tofMax=reductionParameters["tofMax"],
-            tofMin=reductionParameters["tofMin"],
-            version=reductionParameters["version"],
-            wallclockTof=reductionParameters["wallClockTol"],
-            temporalProximity=None,
+            stateId=stateId,
+            tofBin=self.instrumentConfig.delTOverT / self.instrumentConfig.NBins,
```

Comment:
I don't think this is correct.  `delTOverT` means something different inside of `InstrumentConfig`.   Got TOF bin width, we want to use the same value as the d-spacing bin width, which comes from `PixelGroupingParameters`.